flink-yarn/pom.xml

```diff
@@ -113,6 +113,10 @@ under the License.
 						<jvmArg>-Xms128m</jvmArg>
 						<jvmArg>-Xmx512m</jvmArg>
 					</jvmArgs>
+					<args>
+						<arg>-Xmax-classfile-name</arg>
+						<arg>128</arg>
```

Comment:
Does your encrypted file system only support 128 characters?
We should try to set the value as high as possible, otherwise, we might run into [problems](http://scala-language.1934581.n4.nabble.com/Xmax-classfile-name-td4637574.html)